kimchi/wasm/src/plonk_verifier_index.rs

```diff
@@ -219,12 +520,14 @@ macro_rules! impl_verification_key {
                 #[wasm_bindgen(skip)]
                 pub evals: WasmPlonkVerificationEvals,
                 pub shifts: WasmShifts,
-                // TODO: add lookup index field
+                #[wasm_bindgen(skip)]
+                pub lookup_index: Option<WasmLookupVerifier>
```

Comment:
I think JavaScript accepts not passing the parameter to the constructor, see `caml_plonk_verifier_index_to_rust` as it will set it to undefined (?). I would add a TODO in bindings.js.

Reply:
yes if you pass fewer parameters in JS then `undefined` is passed instead

Reply:
This is now fixed